workitem/workitem_repository.go

```diff
@@ -833,6 +850,10 @@ func (r *GormWorkItemRepository) GetCountsForIteration(ctx context.Context, itr
 	db = r.db.Raw(query)
 	db.Scan(&res)
 	if db.Error != nil {
+		log.Error(ctx, map[string]interface{}{
+			"iterationIDs`": whereClause,
```

Comment:
done in https://github.com/almighty/almighty-core/pull/1151/commits/f11d893ea23493f2a73f28b4fed578a77331e80c